### DISCUSSION TITLE
Limitation of eBook generation

### DISCUSSION THREAD

**Discussion by dtctdti9756443** [typemill/typemill#498] [General]
Hi,

Is Typemill has the limit for PDF generation by [eBook Studio] when I selected **layout = [manual]**? Existing we only generate 23 pages in Typemill [Reorder Pages : 7]. However, the full contents should be around 40 pages [Reorder Pages : 9]. 

Tried to use [reset/refresh navigator] but seems not working. Any idea?

- OS : Windows 10 Pro
- Chrome Browser (Version 131.0.6778.86 (Official Build) (64-bit))
- Typemill : version 2.10.0
- Plugin : eBooks Version: 2.2.0
- Book layout [manual]

Remark: If book layout selected [report], the issue is gone.

**Comment by trendschau** [comment]
Please try it again with version 2.2.1 of the ebook plugin, it contains a fix that probably solves your problem: https://plugins.typemill.net/ebooks

You can also try to use Firefox. It is not officially supported by the script that the ebook plugin uses, but in my experience it often works better than chrome.

**Reply by dtctdti9756443** [reply]
Thank you for your reply, and will get back to you @~@
